src/sc_crawler/vendors/_aws.py

```
@@ -1341,20 +1346,413 @@ def inventory_ipv4_prices(vendor):
+    )
+    with ThreadPoolExecutor(max_workers=16) as executor:
+        results = list(
+            executor.map(
+                _describe_orderable_db_instance_options_for_class_with_progress,
+                repeat(regions),
+                database_ids,
+                repeat(vendor),
+            )
+        )
+    vendor.progress_tracker.hide_task()
+    return dict(zip(database_ids, results))
```

Ruff (0.15.21)

[warning] `zip()` without an explicit `strict=` parameter

Add explicit value for parameter `strict=`

(B905)